utilities/src/main/java/io/syndesis/qe/templates/SyndesisTemplate.java

```diff
@@ -82,21 +82,21 @@ private static void deployUsingOperator() {
         TodoUtils.createDefaultRouteForTodo("todo2", "/");
     }
 
-    public static Map<String, Object> getDeployedCr(String namespace, String name) {
-        return getSyndesisCrClient().get(TestConfiguration.openShiftNamespace(), name);
+    public static Map<String, Object> getDeployedCr() {
+        return getSyndesisCrClient().get(TestConfiguration.openShiftNamespace(), "app");
     }
 
-    public static Map<String, Object> editCr(String namespace, String name, Map<String, Object> cr) throws IOException {
-        return SyndesisTemplate.getSyndesisCrClient().edit(namespace, name, cr);
+    public static Map<String, Object> editCr(Map<String, Object> cr) throws IOException {
+        return SyndesisTemplate.getSyndesisCrClient().edit(TestConfiguration.openShiftNamespace(), "app", cr);
     }
 
-    public static void deleteCr(String namespace, String name) {
-        getSyndesisCrClient().delete(namespace, name);
+    public static void deleteCr() {
+        getSyndesisCrClient().delete(TestConfiguration.openShiftNamespace(), "app");
     }
 
-    public static Set<String> getCrNames(String namespace) {
+    public static Set<String> getCrNames() {
         final Set<String> names = new HashSet<>();
-        Map<String, Object> crs = getSyndesisCrClient().list(namespace);
+        Map<String, Object> crs = getSyndesisCrClient().list(TestConfiguration.openShiftNamespace());
```

Comment:
@avano we agreed with @jsafarik that hard-coding the "app" name and always getting the namespace from the test config is ok here, wdyt?

Reply:
And I obviously don't know how the new multiline comments work, this should have been for all the changes in this file

Reply:
maybe I would extract it to variable or constant, but other than that I have no objections

Reply:
Changed it to use a constant

Reply:
I know, but `This pull request is still a work in progress` :dancer:

Reply:
yup, didn't noticed at first, thanks :smile: